src/components/RolesAndSkills.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import { Input, Radio, Form, Select, Option, InputNumber, Button } from 'antd';
+
+/* eslint-disable no-template-curly-in-string */
+
+const validateMessages = {
+  required: '${label} is required!',
+};
+
+/* eslint-disable no-template-curly-in-string */
+
+const RolesAndSkills = () => {
+  const [componentSize, setComponentSize] = React.useState('default');
+
+  const onFormLayoutChange = ({ size }) => {
+    setComponentSize(size);
+//
+  const[form] = Form.useForm();
+  const [requiredMark, setRequiredMarkType] = useState('optional');
+
+  const onRequiredTypeChange = ({ requiredMarkValue }) => {
+    setRequiredMarkType(requiredMarkValue);
+  };
+  };
+
+  return (
+    <Form
+      labelCol={{ span: 6 }}
+      wrapperCol={{ span: 14 }}
+      name="roles_n_skills"
+      validateMessages={validateMessages}
+      size={componentSize}
+      onValuesChange={onFormLayoutChange}
+    >
+      <Form.Item
+        name="roles-interested-in"
+        label="Roles Interested In"
+        rules={[
+          {
+            required: true,
+            message: validateMessages,
+          },
+        ]}
+      >
+        <Checkbox.Group>
+          <Row>
+            <Col span={6}>
+              <Checkbox
+              value="item01"
+              style={{
+                lineHeight: '32px',
+              }}
+              >
+                item01
+              </Checkbox>
+            </Col>
+            <Col span={6}>
```

Comment:
🚫 **[eslint]** <[react/jsx-no-undef](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/jsx-no-undef.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Col' is not defined.